manifests/globals.pp

```diff
@@ -236,7 +236,7 @@
         /12\.0/     => '93',
         /12\.[1-3]/ => '94',
         /12\.[4-5]/ => '12',
-        /15\.[0-9]/ => '14',
+        /15\.[0-9]/ => '16',
```

Comment:
is that a backwards-incompatible change?

Reply:
This is basically setting the postgresql-server default version we use to test. It is setting it to 14 for SLES-15, but currently there is no version 14 in the SLES-15 repos from where we are installing it.
So, I bumped it up to 16 (which is available), ran the tests and they passed.